articles/connections/database/mysql.md

```diff
@@ -79,6 +79,10 @@ This script connects to a **MySQL** database and executes a query to retrieve th
 
 If you are using [IBM's DB2](https://www.ibm.com/analytics/us/en/technology/db2/) product, [click here](/connections/database/db2-script) for a sample login script.
 
+::: panel-info
+You can use the `auth0-custom-db-testharness` [library](https://www.npmjs.com/package/auth0-custom-db-testharness) to deploy, execute, and test the output of Custom DB Scripts using a Webtask sandbox environment.
+:::
+
```

Comment:
I don't know why it does that. I noticed you removed some of my titles, so I stopped adding them.

Alerts are good--I'll start using those instead of panels.